debugdialog.html

```diff
@@ -172,6 +172,15 @@
     return attributes
 }
 
+function getFeatureVariableType(featureKey, variableKey) {
+    let feature = optimizelyClientInstance.getOptimizelyConfig().featuresMap[featureKey];
+    if (feature === undefined) {
+        return null;
+    }
+    var vmap = feature.variablesMap
+    return vmap[variableKey].type
+}
```

Comment:
what if vmap[variableKey] does not exist will it throw or pass back undefined?  couldn't we add vmap[variableKey]?.type

Reply:
@thomaszurkan-optimizely you are right, it will pass back undefined, so we can add optional chaining as you suggested to avoid exceptions.